Playbooks/playbook-Account_Enrichment_-_Generic.yml

```diff
@@ -0,0 +1,221 @@
+id: 51af557d-bd76-483c-81b8-658c4a1e09e9
```

Comment:
Id should be the playbook name

Reply:
Correction: playbook id should be lowercase and with `_` instead of spaces

i.e `account_enrichment_-_Generic`